src/EmailEditor.js

```diff
@@ -1,15 +1,25 @@
-import React, { useCallback, useEffect } from "react";
-const EDITOR_HOST = "https://console.ravenapp.dev/email-editor";
+import React, {
+  forwardRef,
+  useCallback,
+  useEffect,
+  useImperativeHandle,
+} from "react";
+
 var prevCallback = null;
 
-export default function EmailEditor({
-  className,
-  state,
-  onEditorLoad,
-  triggerFetch,
-  onFetched,
-  ...rest
-}) {
+function EmailEditor(
+  {
+    className,
+    state,
+    onEditorLoad,
+    triggerFetch,
+    onFetched,
+    forwardedRef,
+    EDITOR_HOST = "https://console.ravenapp.dev/email-editor",
```

Comment:
variable format should be editorHost = DEFAULT_EDITOR_HOST. declare DEFAULT_EDITOR_HOST as a constant or in an env file. go with env file instead

Reply:
triggerFetch is not required